src/style.css

```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+@font-face {
+  font-family: "DefaultHover";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300px;
+}
+
+@font-face {
+  font-family: "focus";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300px;
+}
+
+.form_big {
+  position: relative;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+  display: block;
+}
+
+.input {
+  height: 70px;
+  padding-left: 62px;
+  width: 100%;
+  display: block;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-family: "DefaultHover";
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  font-family: "focus";
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.form {
+  position: relative;
```

Comment:
no need to repeat styles
check this point
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/102411130/231968469-7f386c3c-daa3-45a6-9371-edda6c7af598.png">